helm-chart/splunk-kubernetes-metrics/values.yaml

```diff
@@ -109,3 +109,20 @@ tolerations:
 
 
 affinity: {}
+
+# = Kubernetes Connection Configs =
+kubernetes:
+  # the URL for calling kubernetes API, by default it will be read from the environment variables
+  url:
```

Comment:
Redundant parameter which is not used anywhere, remove this.

Reply:
done